pom.xml

```diff
@@ -468,7 +468,7 @@
     <hamcrest.version>2.2</hamcrest.version>
     <commons-cli.version>1.4</commons-cli.version>
     <netty.version>4.1.59.Final</netty.version>
-    <jetty.version>9.4.35.v20201120</jetty.version>
+    <jetty.version>9.4.38.v20210224</jetty.version>
     <jackson.version>2.10.5.1</jackson.version>
     <jline.version>2.14.6</jline.version>
     <snappy.version>1.1.7.7</snappy.version>
```

Comment:
> zookeeper-server/src/main/resources/lib/jetty-client-9.4.38.v20210224.LICENSE.txt
> File renamed without changes.
In master, generated zookeeper tar does not contain any jetty-client jar in zookeeper lib.
 jetty-client-9.4.38.v20210224.LICENSE.txt can be removed.

Reply:
Ah, interesting.  I went by `zookeeper-server/target/lib`, but you are of course right that the tarball assembly script does not pick that one up.  Fixed!